pkg/system/stat_openbsd.go

```diff
@@ -0,0 +1,17 @@
+// +build openbsd
```

Comment:
I don't think you need the build tag here, unless Go doesn't recognize the `_openbsd` prefix in the file name.